django-server/server/api/models.py

```diff
@@ -52,7 +71,8 @@ def __str__(self):
     def save(self, *args, **kwargs):
         curr_reviews = FixedMenuOptionReview.objects.count()
         if not self.pk:
-            FixedMenuOption.objects.filter(pk=self.option.pk).update(avg_review_stars=self.calculate_new_avg_review(F('avg_review_stars'), curr_reviews))
```

Comment:
pep8 suggest line was too long

Reply:
@Rhantolq for me you can split this line into two